obspy/core/tests/test_utcdatetime.py

```diff
@@ -16,6 +16,11 @@ class UTCDateTimeTestCase(unittest.TestCase):
     """
     Test suite for obspy.core.utcdatetime.UTCDateTime.
     """
+
+    close_timestamps = [1515174511.1984465, 1515174511.1984463,
+                        1515174511.1984460, 1515174511.1984458]
+    close_utc = [UTCDateTime(x) for x in close_timestamps]
```

Comment:
I really don't like having this definition somewhere else - if possibile I write unit test so that they are self containing - without the need to jump somewhere else in the file in order to understand the test case - but its up to you ;)

Reply:
I usually tend to favor having to jump around rather than repeating myself (there are 3 test cases that use these). Admittedly, they should be tuples and not lists to avoid the possibility of mutation from one test to the next...

On the other hand, consistency is probably more important. I will go ahead and change this like you suggest because that is how the other test cases are.